src/js/binaryen.js-extended.js

```diff
@@ -0,0 +1,56 @@
+// Extended API
+
+// Makes the internal buffer iterable so we can use Uint8Array.from
+if (typeof Symbol !== 'undefined' && Symbol.iterator) {
```

Comment:
i'm having a hard time understanding this... what is `Symbol`? what does the code do with it?